version.tf

```diff
@@ -11,5 +11,9 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = ">= 1.44.3"
     }
+    time = {
+      source  = "hashicorp/time"
+      version = "0.8.0"
```

Comment:
@surajsbharadwaj you should probably of used a range here to make the module flexible. I would suggest creating a new PR to change this to `>= 0.8.0`